### PR TITLE
Markdown Test Branch

### PR DESCRIPTION
**Pull request by rtisma**

### CODE REVIEW

File: file.txt [hunk 1]
```diff
@@ -1 +1,4 @@
 Example file
+
+Add a markdown comment here
```
[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:4]: ignore

File: file.txt [hunk 2]
```diff
@@ -1 +1,4 @@
...
+Add a markdown code snippet here
```
[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:5]: ignore

File: file.txt [hunk 3]
```diff
@@ -1 +1,5 @@
 Example file
+
+
+Add a markdown comment here
```
[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:4]: do something

File: file.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:5]: hello

File: file.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:4]: hello

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:4]: @staging-sonatype-lift hello

File: file.txt [hunk 4]
```diff
@@ -1 +1,7 @@
 Example file
+
+
+
+
+Add a markdown comment here
+Add a markdown code snippet here
```
[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:5]: @staging-sonatype-lift help

[the remark this answers is not in the document]
Comment (ghost) [reply file.txt:5]: I’m LiftBot and I automatically analyze new code in code review, and comment when I find potential bugs. I also accept comments as commands. Just `@sonatype-lift` followed by the command: **ignore** to mark as false positive, **unignore** to undo, and **help** to see this message. [Click here](https://github.com/apps/sonatype-lift/installations/new) to add LiftBot to another repo.

File: file.txt [hunk 5]
```diff
@@ -1 +1,7 @@
 Example file
+
+
+
+
+Add a markdown comment here
```
[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:4]: ignore @staging-sonatype-lift

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:4]: jkhlkjlkj  @staging-sonatype-lift ignore

File: file.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:4]: @sonatype-lift unignore

File: file.txt (hunk 5 above)
[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:4]: @sonatype-lift unignore

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:4]: sdflkjsdflkjsdflkj @sonatype-lift ignore

[the remark this answers is not in the document]
Comment (ghost) [reply file.txt:4]: I've recorded this as ignored for this pull request. If you change your mind, just comment `@sonatype-lift unignore`.

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:4]: @sonatype-lift unignore

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:4]: blah blah ignore @sonatype-lift

File: file.txt (hunk 4 above)
[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:5]: IgNorE

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:5]: ignore bug

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:5]: unignore bug

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:5]: unignore

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:5]: @sonatype-lift ignore buggg

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:5]: @sonatype-lift unignore

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:5]: @sonatype-lift  not a bug is this

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:5]: unignore

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:5]: @sonatype-lift please ignore

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:5]: unignore @sonatype-lift

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:5]: @sonatype-liftignore

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:5]: ignore @sonatype-lift

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:5]: @ sonatype-lift unignore

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:5]: @yosonatype-lift unignore

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:5]: unignore @sonatype-lift

[the remark this answers is not in the document]
Comment (rtisma) [reply file.txt:5]: ignore @sonatype-lift

**Pull request closed without merging**